pkg/provider/data_install_test.go

```diff
@@ -97,6 +98,10 @@ func TestAccDataInstall_basic(t *testing.T) {
 				Config:      testAccDataInstallWithArg("version", "foo"),
 				ExpectError: regexp.MustCompile("\"version\" must either be latest or have the prefix 'v', got: foo"),
 			},
+			{
+				Config:      testAccDataInstallWithArg("baseurl", "http://www.example.org"),
+				ExpectError: regexp.MustCompile("Error: failed to download manifests.tar.gz from http://www.example.org/download/v0.26.2/manifests.tar.gz, status: 404 Not Found"),
```

Comment:
Having a specific version here would require updating it ever time there's an update.
Let's just check for the first part of the error, something like:
```suggestion
				ExpectError: regexp.MustCompile("Error: failed to download manifests.tar.gz"),
```

Reply:
That sounds much better, thanks!